kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -25,423 +24,474 @@
+    
+    ///@{
 
     /**
-    operation to predict the solution ... if it is not called a trivial predictor is used in which the
-    values of the solution step of interest are assumed equal to the old values
+     * Operation to predict the solution ... if it is not called a trivial predictor is used in which the
+     * values of the solution step of interest are assumed equal to the old values
      */
     virtual void Predict()
     {
     }
 
     /**
-    Initialization of member variables and prior operations
+     * Initialization of member variables and prior operations
      */
     virtual void Initialize()
     {
+        KRATOS_ERROR << "You are calling to the base class method Initialize, please define in you derived class the method" << std::endl;
     }
 
     /**
-    the problem of interest is solved.
-	This function calls sequentially: Initialize(), InitializeSolutionStep(), Predict(), SolveSolutionStep() and FinalizeSolutionStep().
-	All those functions can otherwise be called separately.
+     * The problem of interest is solved.
+     * This function calls sequentially: Initialize(), InitializeSolutionStep(), Predict(), SolveSolutionStep() and FinalizeSolutionStep().
+     * All those functions can otherwise be called separately.
      */
     virtual double Solve()
     {
-		Initialize();
-		InitializeSolutionStep();
-		Predict();
-		SolveSolutionStep();
-		FinalizeSolutionStep();
-        return 0.00;
+        Initialize();
+        InitializeSolutionStep();
+        Predict();
+        SolveSolutionStep();
+        FinalizeSolutionStep();
+        
+        return 0.0; 
+        
     }
 
     /**
-    clears the internal storage
+     * Clears the internal storage
      */
     virtual void Clear()
     {
+        KRATOS_ERROR << "You are calling to the base class method Clear, please define in you derived class the method" << std::endl;
     }
 
     /**
-    this should be considered as a "post solution" convergence check which is useful for coupled analysis
-    - the convergence criteria used is the one used inside the "solve" step
+     * This should be considered as a "post solution" convergence check which is useful for coupled analysis
+     * - the convergence criteria used is the one used inside the "solve" step
      */
     virtual bool IsConverged()
     {
         return true;
     }
 
     /**
-    this operations should be called before printing the results when non trivial results (e.g. stresses)
-    need to be calculated given the solution of the step
-
-      This operations should be called only when needed, before printing as it can involve a non negligible cost
+     * This operations should be called before printing the results when non trivial results (e.g. stresses)
+     * need to be calculated given the solution of the step
+     * This operations should be called only when needed, before printing as it can involve a non negligible cost
      */
     virtual void CalculateOutputData()
```

Comment:
i dare say it was not there, or at least not visible in github when i did my revision. Whatever, i accepted your commit